### PR TITLE
feat: Optional max width for Guidance Block

### PR DESCRIPTION
**Pull request by KasiaMisirli**
# Objective
Max width wasn't option, my pr is making it optional.

# Motivation and Context
https://cultureamp.atlassian.net/jira/software/projects/PMC/boards/103?selectedIssue=PMC-943

# Screenshots (if appropriate)
Screen Shot 2021-04-07 at 11 21 17

# Checklist
- [x] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [X] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

File: draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx [hunk 1]
```diff
@@ -120,11 +129,15 @@ class GuidanceBlock extends React.Component<
...
-      [styles.prominent]: this.props.variant === "prominent",
-    })
+  bannerClassName(noMaxWidth): string {
+    return classnames(
+      styles.banner,
+      {
+        [styles.hidden]: this.state.hidden,
+        [styles.prominent]: this.props.variant === "prominent",
+      },
+      noMaxWidth && styles.noMaxWidth
```
Comment (ActuallyACat) [inline draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx:139]:
```suggestion
        [styles.prominent]: this.props.variant === "prominent",
        [styles.noMaxWidth]: noMaxWidth
      },
```
nit/ although this works as intended, this is the pattern that we usually use alongside classnames when a style is conditional.

File: draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx [hunk 2]
```diff
@@ -81,6 +81,16 @@ const WithoutActionArrowButton = () => (
   </div>
 )
 
+const WithoutMaxWidth = () => (
+  <div style={{ display: "flex", justifyContent: "center", margin: "100px" }}>
+    <GuidanceBlock
+      img={{ src: guidanceBlockImg, alt: "Guidance block" }}
```
Comment (ActuallyACat) [inline draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx:87]: nit/ And not that relevant for this repo - but an empty `alt` tag is useful for screen readers when an image is purely decorative / does not add additional context (like this one): https://www.w3.org/WAI/tutorials/images/decorative/

**Review by ActuallyACat — DISMISSED** [review]

**Comment by ActuallyACat** [comment]
Some nitpicking, but otherwise LGTM.

File: draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx (hunk 2 above)
Comment (KasiaMisirli) [reply draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx:87]: I guess its more of a consideration when the component is actually used in the code base, will make sure to add empty string there!

**Review by ActuallyACat — APPROVED** [review]

**Pull request merged**